openai_helper/ui/app.py

```diff
@@ -15,7 +15,11 @@ def __init__(self, configuration_path: str | Path | None = None):
         super().__init__()
         self.configuration = Configuration(configuration_path)
         self.title("OpenAI Helper")
-        self.attributes("-zoomed", True)
+
+        # Maximize the window on all platforms
+        self.attributes("-zoomed", True)  # For Windows and Linux
+        self.wm_state("zoomed")  # For MacOS
```

Comment:
AFAIK this will still fail on MacOS, maybe even the second line would fail on Windows / Linux. Could we do this conditionally? If the platform is darwin, call the zoomed state, if not, use the attribute instead.

Reply:
good point - changed.